docs/source/install/rhel7.rst

```diff
@@ -0,0 +1,387 @@
+RHEL 7 / CentOS 7
+=================
+
+This guide provides step-by step instructions on installing StackStorm on a single box on RHEL 7/CentOS 7.
+A script `st2bootstrap-el7.sh <https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-el7.sh>`_,
+codifies the instructions below.
+
+.. warning :: Currently BETA! Please try, use and report bugs on
+   `github.com/StackStorm/st2-packages <https://github.com/StackStorm/st2-packages/issues/new>`_.
+   Soon, package-based installation will be
+   the preferred path to installing StackStorm.
+
+.. contents::
+
+
+Minimal installation
+--------------------
+
+Install Dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+Install MongoDB, RabbitMQ, and PostgreSQL.
+
+  .. code-block:: bash
+
+    sudo yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    sudo yum -y install mongodb-server rabbitmq-server
+    sudo systemctl start mongod rabbitmq-server
```

Comment:
2) Yep. I hit that myself. We need two changes - one in docs and another in script. This is tedious. More like a general point. 

1) I don't get you. Are you saying s/systemctl/service? I noticed that in docker instructions myself. I can make that change.